ops/terraform/main.tf

```diff
@@ -295,6 +295,7 @@ resource "google_compute_firewall" "bacalhau_ingress_firewall" {
       "4001",  // ipfs swarm
       "1234",  // bacalhau API
       "1235",  // bacalhau swarm
+      "6001",  // local publisher download
```

Comment:
I think we shouldn't enable this publisher in our public network, but at the same time we need to test it in different environments 🤷‍♂️